packages/shaders/src/index.ts

```diff
@@ -1,5 +1,5 @@
 /** The core Shader Mount class. Pass it a parent element and a fragment shader to get started. */
-export { ShaderMount, isPaperShaderElement } from './shader-mount';
+export { ShaderMount, isPaperShaderElement, DEFAULT_MAX_PIXEL_COUNT } from './shader-mount';
```

Comment:
Let's clean up this export since we don't really need it in the docs, it's a bit random for the end user

Reply:
cool cool - only added it for the slider (which imo was very satisfying to use!)